x-pack/plugin/otel-data/src/yamlRestTest/resources/rest-api-spec/test/20_logs_tests.yml

```diff
@@ -187,3 +187,28 @@ host.name pass-through:
   - length: { hits.hits: 1 }
   - match: { hits.hits.0.fields.resource\.attributes\.host\.name: [ "localhost" ] }
   - match: { hits.hits.0.fields.host\.name: [ "localhost" ] }
+---
+"kubernetes.* -> resource.attributes.k8s.* aliases":
+  - do:
+      bulk:
+        index: logs-generic.otel-default
+        refresh: true
+        body:
+          - create: {}
+          - '{"@timestamp":"2024-07-18T14:48:33.467654000Z","data_stream":{"dataset":"generic.otel","namespace":"default"}, "resource" : { "attributes": { "k8s.container.name": "myContainerName", "k8s.cronjob.name" : "myCronJobName", "k8s.job.name" : "myJobName", "k8s.statefulset.name" : "myStatefulsetName", "k8s.daemonset.name" : "myDaemonsetName", "k8s.replicaset.name" : "myReplicasetName", "k8s.node.uid" : "myNodeUid", "k8s.node.hostname" : "myNodeHostname" } } }'
```

Comment:
Maybe specify this as yaml like in the test above? I guess it's a matter of taste, but I personally find it more readable rather than having a long line of JSON.